spec/order_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'order'
+
+describe Order do
+  
+  let(:dish) { double :dish, :name => "Pizza", :price => 5 }
+  it 'has a default total of 0' do
+    expect(subject.total).to eq 0
```

Comment:
i'm not sure i've understood this test correctly, we shall discuss, but isn't the total supposed to equal 5? or am i missing something?